website/docs/basics/blocks-special-use-cases.md

```diff
@@ -6,5 +6,84 @@ sidebar_label: Special Use Cases
 
 [![docs-source](https://img.shields.io/badge/source-eigthshift--frontend--libs-yellow?style=for-the-badge&logo=javascript&labelColor=2a2a2a)](https://github.com/infinum/eightshift-frontend-libs/tree/develop/blocks/init/src/blocks/)
 
+By special use cases, we mean just that **special**.
+For example, in our clients projects, we saw that sometimes you need to provide something special to your blocks, like changing the block editor block DOM structure or change attributes added inside the inner blocks.
 
-*Coming soon*
+These are some of the examples that we found that we would like to share with you. Here we will list all of them and gradually add more.
+
+## Change innerBlock attributes
+
+**Usage:**
+
+- Change attributes in the block editor and in the frontend.
+- Set attributes in the database.
+
+**Use Case:**
+
+Let's say you have a carousel block, and you have a generic image block that you can use as a standalone block. You create a carousel block and set in its manifest limitation to show only the image block to be inserted as an inner block. This is easy to do; you just add your image block to an `allowedBlocks` array inside the carousel block, and everything works. 
+
+But now you have some default attributes set in that image block that you don't want to be set if the block is added using the innerBlock in the carousel. You can change that using our [overrideInnerBlockAttributes](helpers-javascript#overrideinnerblockattributes) helper. Just add this helper to your block entry point, add the attributes you want to change, and you are all set. 
+
+Keep in mind that this will be applied to all innerBlocks inside your carousel block. If you want to limit this to only a specific block, you will need to fiddle around with that block's conditions (you can get that specific block name from the props).
+
+Next time you add your image to the carousel, it will automatically change those attributes.
+
+> Important note: If you for some reason copy the inner block from the carousel to the top-level editor, this helper will not fire and not change back to original attributes. You have to manually remove those attributes from the Code editor in the admin.
+
+**Implementation**
+
+Here you can see our example of this helper used in the [carousel block](https://github.com/infinum/eightshift-frontend-libs/blob/develop/blocks/init/src/Blocks/custom/carousel/carousel-block.js).
+
+## Provide simple wrapper to all innerBlock
+
+**Usage:**
+- Change attributes in the block editor and in the frontend.
+- Set attributes in the database.
+
+**Use Case:**
+
+All of our blocks come with the wrapper implemented, and this is awesome, but sometimes you just don't want to use the wrapper, and you want only to use the simple version of the wrapper. You can provide the attributes overrides in your block, and you are good to go.
```

Comment:
Can you add the link to a page explaning what is a simple wrapper here? Can be anchored to `simple version of the wrapper` text.